src/components/drawing/index.js

```diff
@@ -579,16 +579,17 @@ drawing.makeTester = function(gd) {
         tester.node()._cache = {};
     }
 
-    gd._tester = tester;
-    gd._testref = testref;
+    drawing.tester = gd._tester = tester;
+    drawing.testref = gd._testref = testref;
```

Comment:
@alexcjohnson can we agree on fixing the child window behavior in an another PR?

I'm ok leaving this PR as is or pushing two more commits replacing `gd._tester` -> `Drawing.tester` and `gd._testref` -> `Drawing.testref`.